Training/Program.cs

```diff
@@ -1,7 +1,34 @@
-namespace Training {
-    internal class Program {
-        static void Main (string[] args) {
-            Console.WriteLine ("Hello, World!");
-        }
-    }
+//Password validity checker which checks if the input password is valid or not
+while (true) {
+   Console.Write ("Password: ");
+   string password = Console.ReadLine ();
+   if (StrongPassword (password)) {
+      Console.WriteLine ("Accepted");
+      break;
+   } else {
+      Console.WriteLine ("\nInvalid password. Try a new password.");
+   }
+  static bool StrongPassword (string password) {
+      if (!password.Any (char.IsUpper)) {
+         Console.WriteLine ("Password must start with a Uppercase");
+         return false;
+      }
+      if (!password.Any (char.IsDigit)) {
+         Console.WriteLine ("Password must atleast contain a digit");
+         return false;
+      }
+      if (!password.Any (char.IsLower)) {
+         Console.WriteLine ("Password must contain atleast one lowercase");

```

Comment:
The message should be: "Password must contain at least one lowercase letter".
While conveying some message to the user, see that it doesn't have any errors as per the language: 
For instance, it should be "at least" instead of "atleast".